src/api/services/PetService.ts

```diff
@@ -1,30 +1,33 @@
+import { Connection } from 'typeorm';
 import { Pet } from '../../models/Pet';
 import { PetRepository } from '../repositories/PetRepository';
 
 export class PetService {
     private repository: PetRepository;
 
-    constructor() {
-        this.repository = new PetRepository();
+    constructor(connection: Connection) {
+        this.repository = connection.getCustomRepository(PetRepository);
     }
 
-    public insert(pet: Pet): any {
-        return this.repository.insert(pet);
+    public async insert(pet: Pet): Promise<Pet> {
+        return this.repository.save(pet);
```

Comment:
@bigLucas 
This `return` _(line 13)_ doesn't need to have an `await` ?

Reply:
@Aristevao 
Yes, I'll change it.

Reply:
@Aristevao 
Please **update** this branch, you can make a `commit` in this branch and make a `push` with the change.
Do you know how you can do this?

Reply:
@bigLucas 
Are the steps below?
```
git fetch --all
git checkout M-0008_refactoring_repository_layer_of_pet_entity
git pull
~ Make the changes
git add .
git commit -m "<message>"
git push
```

Reply:
Yes.
```shell
git push origin M-000...branch_name...
```